apps/store/src/services/priceIntent/PriceIntentService.ts

```diff
@@ -70,7 +65,11 @@ export class PriceIntentService {
     return await this.create({ productName })
   }
 
-  public reset() {
-    this.persister.reset()
+  private priceIntentKey(productName: string) {
```

Comment:
It could be useful to be explicit whether it's a setter method or getter:

```suggestion
  private getPriceIntentKey(productName: string) {
```

Reply:
good point, especially since it takes a param